build.clj

```diff
@@ -0,0 +1,71 @@
+(ns build
+  (:require [clojure.tools.build.api :as b]
+            [deps-deploy.deps-deploy :as dd]
+            [borkdude.gh-release-artifact :as gh]))
+
+(def lib 'timokramer/datahike)
```

Comment:
should this be `io.replikativ/datahike`?

Reply:
right :)